### DISCUSSION TITLE
[SOLVED] Prellenado de propiedades de usuario dejó de funcionar

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#75] [4. Bug Reports] [answered]
Buenas,

Tengo una question de propiedades donde antes sí servía el prellenado y ahora no con los mismos datos de entrada.
```
function run () { 
  const commands = []; 
  const user = context['user#me']; 
  commands.push({ cmd: 'SET_RESPONSES', value: user.properties[1] });
  return commands;
}
```
El usuario tiene en `properties[1]` la propiedad de ese tipo. Esto funcionaba hace una semana.

Saludos!

**Comment by acedward** [comment, marked as answer]
@SeIturra de que tipo de Question, y que tipo de QuestionExec es este?

**Reply by seiturra** [reply]
propiedades y preload

**Reply by seiturra** [reply]
"contentType" : "application/vnd.cotalker.survey+property", 
    "exec" : {
        "preload" : {
            "src" : "function run () { const commands = []; const user = context['user#me']; commands.push({ cmd: 'SET_RESPONSES', value: user.properties[1] });return commands;}", 
            "context" : "property#user,user#me"
        }
    }

**Reply by acedward** [reply]
@guillevalin  parece un bug - parece una configuración legal -

**Reply by guillevalin** [reply]
Tuvimos algunos bugs con los formularios que debieron solucionarse con la última versión en productivo (15.5.4) sigue pasando?

**Reply by seiturra** [reply]
Solved!
@guillevalin y @acedward ahora no sucede.
Gracias a los dos!
